src/Client/WorldMapWindow.java

```diff
@@ -1430,8 +1430,10 @@ public void showWorldMapWindow() {
     if (!frame.isVisible()) {
       searchText = "";
       updateMapRender();
+      frame.setVisible(true);
```

Comment:
Same here actually. showWorldMapWindow() should just show it & there would want to be a toggleWorldMapWindow() function.

Reply:
Agreed - here, I thought it would be appropriate for both callers to invoke the toggle, ie. both the keybind as well as the icon click. Same thought on the keybind as the other comment.